src/Conda.jl

```diff
@@ -137,34 +192,40 @@ function _install_conda(force=false)
             end
         end
         Conda.add_channel("defaults")
+        # Update conda because conda 4.0 is needed and miniconda download installs only 3.9
+        run(_set_conda_env(`$(Conda.conda) update -y conda`))
+    end
+    if !isdir(prefix(env))
+        # conda doesn't allow totally empty environments. using zlib as the default package
+        run(_set_conda_env(`$(conda_bin(rootenv)) create -y -p $(prefix(env)) zlib`, rootenv))
     end
 end
 
 "Install a new package."
-function add(pkg::AbstractString)
-    _install_conda()
-    run(_set_conda_env(`$conda install -y $pkg`))
+function add(pkg::AbstractString, env::Environment=rootenv)
+    _install_conda(env)
+    run(_set_conda_env(`$(conda_bin(env)) install -y $pkg`, env))
 end
 
 "Uninstall a package."
-function rm(pkg::AbstractString)
-    _install_conda()
-    run(_set_conda_env(`$conda remove -y $pkg`))
+function rm(pkg::AbstractString, env::Environment=rootenv)
+    _install_conda(env)
+    run(_set_conda_env(`$(conda_bin(env)) remove -y $pkg`, env))
 end
 
 "Update all installed packages."
-function update()
-    _install_conda()
+function update(env::Environment=rootenv)
+    _install_conda(env)
     for package in _installed_packages()
-        run(_set_conda_env(`$conda update -y $package`))
+        run(_set_conda_env(`$(conda_bin(env)) update -y $package`, env))
```

Comment:
Shall I give all the packages as one command? Eg: `conda update -y numpy scipy`
Advantage would be that if conda cannot find a valid set of versions to update to, this will fail.